src/modules/navigation/components/Navigation/Navigation.module.scss

```
@@ -214,6 +214,14 @@ $c-navigation-decoration: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDo
 	&--right {
 		grid-column: 3/4;
 		justify-self: end;
+
+		// Ward: first child element is MaterialRequestCenterButton, which has a blade
+		// the other navigation items pop up through the blade if the following z-index is removed
+		ul {
+			li:first-child {
+				z-index: 5;
```

use the get-z-layer function. The filter layer is equal to 5.
```suggestion
				z-index: z-index: get-z-layer("filter");
```